agent/base

```diff
@@ -208,6 +208,15 @@ function verify_tool_group {
 	return 0
 }
 
+function generate_inventory {
+	export INVENTORY=/tmp/inventory.$$
+	echo "[controller]"
+	echo $HOSTNAME
+	echo "[remote]"
+	ls $tools_group_dir | awk -F@ '/^remote/ {print $2};'
+	trap "rm -f $INVENTORY" QUIT INT EXIT
+}
+
```

Comment:
This should probably be structured a bit differently:

     export INVENTORY=
     trap ...
      function generate_inventory {
            ....
      }

      generate_inventory > $INVENTORY
       ,,,